cli/cmd/plugin/ui/web/tanzu-ui/src/state-management/stores/Store.umc.tsx

```diff
@@ -5,24 +5,14 @@ import React, { createContext, ReactNode, useReducer } from 'react';
 import { STORE_SECTION_FORM } from '../reducers/Form.reducer';
 import { StoreDispatch } from '../../shared/types/types';
 import umcReducer from '../reducers/Wizard.reducer';
+import { UNMANAGED_CLUSTER_FIELDS } from '../../views/unmanaged-cluster/unmanaged-cluster-wizard-page/UnmanagedCluster.constants';
+import { UNMANAGED_DEFAULT_VALUES } from '../../shared/constants/defaults/unmanaged.defaults';
```

Comment:
It's odd to me that we have `UnmanagedCluster.constants` where I would expect (in unmanaged-cluster-wizard-page), but the default values (`unmanaged.defaults`) are in a shared location. What do folks think of moving those defaults to be in the same place as the constants?

Reply:
If we do decide that can I just change it in a future PR?

Reply:
Completely agree, the defaults should sit next to the constants. Christian, I would move the unmanaged cluster defaults in this PR then we can refactor the others in a follow-up PR.